scripts/create_github_release.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+SCRIPT_DIR="$(cd "$(dirname "$0")" && pwd)"
+source "${SCRIPT_DIR}/utils.sh"
+
+set -eo pipefail
+
+[[ -n "${GITHUB_TOKEN}" ]] || die "No GITHUB_TOKEN found."
+
+tag="$1"
+bin_dir="$2"
+[[ -n "${tag}" && -n "${bin_dir}" ]] || die "Usage: $0 <tag> <binary_directory>"
+
+mkdir /tmp/release-artifacts
+for os in darwin linux windows; do
+  bin_name="kube-linter"
+  if [[ "${os}" == "windows" ]]; then
+    bin_name="kube-linter.exe"
```

Comment:
It doesn't make a lot of sense to package a windows EXE in a tgz. Maybe distribute everything as both zip and tgz?

Reply:
Kubernetes puts Windows in a tar.gz: https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.18.md#client-binaries But yeah, no harm in distributing everything as zip + tgz.